draft-specifications/CapTP Specification.md

```diff
@@ -689,6 +689,90 @@ resolver when a resolution is available to the `listen-desc` object. If a
 resolution is already available, the resolver provided in `listen-desc` MUST be
 fulfilled or broken.
 
+## `op:get`
+
+`op:get` requests the value for the named field of an eventually settled
+[Struct](Model.md#Struct).  The get operation follows promise resolutions,
+inheriting the rejection reason of any intermediate rejected promise.
+The operation rejects the answer if the ultimate fulfillment of the receiver
+is not a struct.
+
+The messages looks like:
+```
+<op:get <receiver-pos>       ; <desc:answer | desc:import-promise>
+        <field-name>         ; String
+        <new-anser-pos>      ; Positive Integer
+```
+
+> The `op:get` operation allows a sender to pipeline messages to a
+> [Target](Model.md#Target) that is deeply embedded in one or more enveloping
+> structs.
+> For cases where the receiver of a get operation is an answer slot with no
+> listeners, sending `op:get` obviates the transmission of uninteresting fields
+> of a potentially large Struct.
+
+### Sending
+#### `receiver-pos`
+This must be the `desc:answer` or a`desc:import-promise` value which
```

Comment:
I deviated here from the “should” verb in `op:pick`, so if this is deemed more appropriate generally, I can follow-up with another editorial change to just make that language consistent.